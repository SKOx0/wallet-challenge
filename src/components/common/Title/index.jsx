import styled from 'react-emotion';

export default styled('p')`
    color: ${(props) => props.color};
`;
