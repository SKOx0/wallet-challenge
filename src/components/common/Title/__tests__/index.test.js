import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Title from '../index';

describe('<Title />', () => {
  it('deve ser igual ao snapshot', () => {
    const renderedComponent = shallow(<Title color="#ffffff" />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
