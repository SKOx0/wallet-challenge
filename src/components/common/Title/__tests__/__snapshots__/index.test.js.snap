// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> deve ser igual ao snapshot 1`] = `
.emotion-0 {
  color: #ffffff;
}

<p
  className="emotion-0"
  color="#ffffff"
/>
`;
